### PR TITLE
Additional changes for #3500

### PR DESCRIPTION
**Pull request by justinharrell**
Some white-space cleanup and a test for the #3500 fix

### CODE REVIEW

**Comment by wimrijnders** [comment]
Nice and concise as usual. I really don't want to take the level of your work for granted, it's consistently exceptional. I'll do an actual run with the new unit test just to be sure. Maybe it will also give me some ideas to comment about.

**Comment by wimrijnders** [comment]
Giving it the label 'Confirmed Bug' because that is what the originating PR #3500 has, and this can be viewed as a fix on that.

**Comment by wimrijnders** [comment]
Unit test works as advertised. Please tolerate my niggles.

File: test/Network.test.js [hunk 1]
```diff
@@ -610,6 +610,37 @@ describe('Edge', function () {
...
+    // create a network
+    var data = {
+        nodes: nodes,
+        edges: edges
+    };    
+
+    var container = document.getElementById('mynetwork');
+    var network = new vis.Network(container, data);
+
+    nodes.remove(node2.id);
```
Comment (wimrijnders) [inline test/Network.test.js]: Assert here that the edge has indeed dropped out of `network.body.edges`.

File: test/Network.test.js [hunk 2]
```diff
@@ -610,6 +610,37 @@ describe('Edge', function () {
...
+    nodes.add(node2);
+    
+    var edges = network.body.edges;
+    var foundEdge = edges[edge.id];
+    
+    assert.ok(foundEdge!==undefined, "edge is missing from state cache");
+
```
Comment (wimrijnders) [inline test/Network.test.js]: Remove empty line.

**Comment by justinharrell** [comment]
No problem, done, I got lazy and didn't cleanup my test code before pushing, also made the comments a little better.

**Comment by wimrijnders** [comment]
All I can say is, apparently your definition of 'laziness' conflicts with mine. Thanks for the good work.

**Pull request merged**